e2e/test/scenarios/question-reproductions/reproductions-3.cy.spec.js

```diff
@@ -1584,47 +1585,40 @@ describe("issue 44668", () => {
 });
 
 // TODO: unskip when metabase#44974 is fixed
-describe.skip(
-  "issue 44974",
-  {
-    tags: ["@external"],
-  },
-  () => {
-    beforeEach(() => {
-      restore("postgres-writable");
-      cy.signInAsAdmin();
-    });
-    it("should not be possible to join with a table or question which is not in the same database (metabase#44974)", () => {
-      startNewQuestion();
-      entityPickerModal().within(() => {
-        entityPickerModalTab("Tables").click();
-        cy.findByText("Sample Database").click();
-        cy.findByText("Orders").click();
-      });
-
-      cy.button("Visualize").click();
-      cy.button("Save").click();
-      modal().button("Save").click();
+describe.skip("issue 44974", () => {
+  const PG_DB_ID = 2;
 
-      cy.wait(300);
-      modal().button("Not now").click();
+  beforeEach(() => {
+    restore("postgres-12");
+    cy.signInAsAdmin();
+  });
 
-      startNewQuestion();
-      entityPickerModal().within(() => {
-        entityPickerModalTab("Tables").click();
-        cy.findByText("Writable Postgres12").click();
-        cy.findByText("Scoreboard Actions").click();
-      });
+  it("should not be possible to join with a question from recents which is not in the same database (metabase#44974)", () => {
```

Comment:
This begs the question - does it really filter out entities from other databases in all other tabs as well?

```suggestion
  it("entity picker should not offer to join with a table or a question from a different database (metabase#44974)", () => {
```

Reply:
Sorry if I'm being nitpicky here, but in this particular case semantics matter.
We know it shouldn't be possible to do this.

But the issue is about a certain UI element presenting a user with an action that they cannot execute.